Makefile

```diff
@@ -5,6 +5,12 @@ default: install
 help: # with thanks to Ben Rady
 	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 
+ifeq ($(shell which g++-11),)
```

Comment:
Axed the CC/CXX business. Now just detect g++-11 and fallback to 10. But also allow someone to pass `TOOLCHAIN` as an arg to `make` if they really want to.